<?xml version="1.0"?>
<launch>

	<rosparam file="$(find floaty_pkg)/config/all_config.yaml"/>


	<!-- Node that read Optitrack data and publishs it over ROS --> 
	<!-- <group ns="NatNet">
		<node name="NatNet_node" pkg="floaty_pkg" type="NatNet_node" 
						required="true" output="screen"/>
	</group> -->


	<!-- Node that logs floaty information published over a specific topic --> 
	<!-- <node name="log_floaty_info_node" pkg="floaty_pkg" type="log_floaty_info_node.py" 
					required="true" output="screen"/> -->


	<!-- Node that runs a service to manage communication with CF over radio --> 
	<!-- <group ns="crazyradio">
		<node name="crazyradio_node" pkg="floaty_pkg" type="crazyradio_node.py" 
					 required="true" output="screen"/>
	</group> -->


	<!-- Node that runs a service to control Windy's maestro motors --> 
	<!-- <group ns="maestro_control">
		<node name="maestro_node" pkg="floaty_pkg" type="maestro_node" 
					 required="true" output="screen"/>
	</group> -->

	
	<!-- Node that runs a service to control Windy's maestro motors --> 
	<group ns="maestro_side_wind_control">

		<node name="maestro_side_wind_node" pkg="floaty_pkg" type="maestro_side_wind_node" 
					 required="true" output="screen"/>
	</group>

</launch>
